kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -5,32 +5,50 @@
       <logo class="logo"/>
       <h1 class="login-text title">{{ $tr('kolibri') }}</h1>
       <form id="login-form" ref="form" @submit.prevent="signIn">
-        <core-textbox
-          :label="$tr('username')"
-          id="username"
-          :placeholder="$tr('enterUsername')"
-          :aria-label="$tr('username')"
-          v-model="username"
-          required
-          autofocus/>
-        <core-textbox
-          :label="$tr('password')"
-          id="password"
-          type="password"
-          :placeholder="$tr('enterPassword')"
-          :aria-label="$tr('password')"
-          v-model="password"
-          autocomplete="current-password"
-          required/>
+        <transition name="textbox">
+          <core-textbox
+            v-if="(!simpleLogin || !passwordMissing)"
+            :label="$tr('username')"
+            id="username"
+            :placeholder="$tr('enterUsername')"
+            :aria-label="$tr('username')"
+            v-model="username"
+            required
+            autofocus
+            @focus="showDropdown = true"
+            @blur="showDropdown = false"
+            @keydown="handleKeyboardNav"/>
+        </transition>
+        <transition name="list">
+          <ul class="suggestions" v-if="simpleLogin && suggestions.length && !uniqueMatch && !passwordMissing"
+            v-show="showDropdown">
+            <ui-autocomplete-suggestion v-for="(suggestion, i) in suggestions"
+              :suggestion="suggestion"
+              @click.native="fillUsername(suggestion)"
+              :class="{ highlighted: highlightedIndex === i }"/>
+          </ul>
+        </transition>
+        <transition name="textbox">
+          <core-textbox
+            :label="$tr('password')"
+            v-if="(!simpleLogin || (simpleLogin && passwordMissing))"
+            id="password"
+            type="password"
+            :placeholder="$tr('enterPassword')"
+            :aria-label="$tr('password')"
+            v-model="password"
+            autocomplete="current-password"
+            required/>
+        </transition>
         <icon-button id="login-btn" :text="$tr('signIn')" :primary="true" type="submit"/>
 
-        <p v-if="loginError" class="sign-in-error">{{ $tr('signInError') }}</p>
+        <p v-if="invalidCredentials" class="sign-in-error">{{ $tr('signInError') }}</p>
       </form>
       <div id="divid-line"></div>
```

Comment:
Might be good to add a bit more margin between the sign in button and the section below.